pkg/basetypes.go

```diff
@@ -192,6 +192,8 @@ type UpkeepPayload struct {
 	CheckData []byte
 	// Trigger is the event that triggered the upkeep to be checked
 	Trigger Trigger
+	// MercuryLookup a flag that determines if a mercury lookup should be performed
+	MercuryLookup bool
```

Comment:
How we communicate `MercuryLookup` on the upkeep payload was a little up in the air; so I opted for the most straightforward approach here, just to get something in place. We can discuss if we want a map to store config values, and where exactly within the payload that map should live if neede 👍

Reply:
Looks like it will work just fine.